src/sage/numerical/interactive_simplex_method.py

```diff
@@ -2657,6 +2674,43 @@ def run_simplex_method(self):
             Entering: $x_{2}$. Leaving: $x_{3}$.
             ...
             The optimal value: $6250$. An optimal solution: $\left(250,\,750\right)$.
+            
+        Using a differnt pivot rule::
```

Comment:
Please run `./sage -tox -e codespell -- src/sage/numerical/interactive_simplex_method.py` and fix any typos